src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -226,6 +228,27 @@ public override void Dispose()
             _node.WorkspaceViewModel.PropertyChanged -= Workspace_PropertyChanged;
         }
 
+        internal void PlaceNodeAutocompleteWindow(object sender, EventArgs e)
+        {
+            var popup = sender as NodeAutoCompleteSearchControl;
+
+            double x;
+            if (PortModel.PortType == PortType.Input)
+            {
+                // Position node autocomplete UI offset left from X position of node by its width.
+                x = _node.X - popup.ActualWidth;
+            }
+            else
+            {
+                // Position node autocomplete UI offset right from X position of node by node width.
+                x = _node.X + _node.NodeModel.Width;
+            }
+            // Position UI down from the top of the node but offset by the node header and against the respective port.
+            var y = _node.Y + NodeModel.NodeHeaderHeight + PortModel.Index * PortModel.Height;
```

Comment:
This way I guess there is no space between?

Reply:
No, there is no space. I can add some constant spacing, just wasn't sure the exact amount. Will add a GIF for folks to review.